source/frontend/src/features/mapSideBar/MapSideBar.tsx

```diff
@@ -94,8 +94,11 @@ const StyledMapSideBar = styled.div<{ sideBarState: IMapSideBarViewState }>`
       } else {
         return `
           min-width: 93rem;
-          margin-left: 0rem;
           max-width: 93rem;
+          @media only screen and (max-width: 1199px) {
+            min-width: 70rem;
```

Comment:
This is arbitrary, but was chosen as it still displays most sidebar screens without needing major changes, while simultaneously displaying enough of the map that the map view is still useable.